extern/ProtoGeometry/ProtoGeometry.XML

```diff
@@ -4486,6 +4486,7 @@
             Some parameters take single value or list - one value per curve.
             </summary>
             <param name="curves">A list of curves to create pipes from</param>
+            <param name="inSmoothMode">Show T-Spline Surface in box or smooth visualization</param>
```

Comment:
@QilongTang I see the order of this param description is wrong. It should be at the very end as per the method signature. Since `TSplineSurface.cs` is not autogenerated, you need only to edit this file in `ProtoGeometry` in LibG.

Reply:
@aparajit-pratap Fixed.